modules/nodes-pods-autoscaling-creating-cpu.adoc

```diff
@@ -70,6 +70,53 @@ $ oc autoscale dc/<dc-name> \// <1>
 <2> Optionally, specify the minimum number of replicas when scaling down.
 <3> Specify the maximum number of replicas when scaling up.
 <4> Specify the target average CPU utilization over all the pods, represented as a percent of requested CPU. If not specified or negative, a default autoscaling policy is used.
++
+[TIP]
+====
+You can alternatively apply the following YAML to scale based on the percent of CPU:
+
+.Sample `HorizontalPodAutoscaler` object for scaling a deployment config on the percent of CPU utilization
```

Comment:
My instinct is that we don't want these block titles, but I we can check with others if you believe they are helpful for readers.